src/hello-world/components/hello-world.tsx

```diff
@@ -1,32 +1,82 @@
 import React, { useState } from 'react';
-import { WithControls, ControlButton, WithStyles, Script } from 'smart-builder-sdk';
+import { WithControls, WithStyles, Script, getAfterFormSubmitScript } from 'smart-builder-sdk';
 import { ComponentProps, WithStylesProps } from 'unbounce-smart-builder-sdk-types';
 
+import manifest from '../../../manifest';
+import { EmptyActions, EmptyState, StyledImg, Wrapper, Text } from '../styled';
 import { ChangeFirstNameModal } from './change-first-name-modal';
 import { Panel } from './control-panel';
+import { OptionsControlButton } from './options-control-button';
 import { getInlineScript, runOnloadMethod } from './script';
 
-export type DataStructure = { firstName: string; lastName: string; styles: { textAlign: string } };
+export type DataStructure = {
+  fullname: string;
+  isButtonSet: boolean;
+  styles: { textAlign: string };
+  tracking: { trackingEnabled: boolean };
+};
 
-const HelloWorld = ({ data, dispatch, className, mode }: ComponentProps<DataStructure, WithStylesProps>) => {
-  const { firstName, lastName } = data;
+const HelloWorld = ({ data, dispatch, className, mode, entityId }: ComponentProps<DataStructure, WithStylesProps>) => {
+  const { fullname, isButtonSet, tracking } = data;
   const [showModal, setShowModal] = useState(false);
+  const buttonId = `myAppButton-${entityId}`;
 
   const updateFirstName = (newFirstName: string) => {
```

Comment:
I would change the name of the function to `updateFullName` and also the parameter to `newFullName` for consistency.

Reply:
Sure! I will do the change 👌🏻